engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/views/activation-keys-table-full.html

```diff
@@ -2,7 +2,7 @@
   <thead>
     <tr bst-table-head>
       <th bst-table-column="name" sortable><span translate>Name</span></th>
-      <th bst-table-column="consumed"><span translate>Consumed</span></th>
+      <th bst-table-column="Host Limit"><span translate>Consumed</span></th>
```

Comment:
I think you want this:
`<th bst-table-column="consumed"><span translate>Host Limit</span></th>`